### ISSUE TITLE
Using a 16-bit texture as EXT_structural_metadata results in false positives

### ISSUE THREAD

**Issue by bjornblissing** [CesiumGS/3d-tiles-validator#354]
When using a greyscale 16-bit PNG texture as `EXT_structural_metadata` the following warnings is emitted:

```json
{
  "type": "TEXTURE_CHANNELS_SIZE_MISMATCH",
  "path": "./12.glb/propertyTextures/0/properties/dhm_u16",
  "message": "The property 'dhm_u16' has the component type UINT16, with a size of 2 bytes, and the type SCALAR with 1 components, resulting in 2 bytes per element, but the number of channels in the property texture property was 1",
  "severity": "WARNING"
}
```

The definition of the metadata looks like this:
```json
"extensions": {
  "EXT_structural_metadata": {
    "schema": {
      "id": "ID9F119BA6_1BA3_4A1B_BD80_4F76BB3A02D6",
      "classes": {
        "dhm_u16": {
          "properties": {
            "dhm_u16": {
              "type": "SCALAR",
              "componentType": "UINT16",
              "normalized": true,
              "noData": 65535
            }
          }
        }
      }
    },
    "propertyTextures": [{
        "class": "dhm_u16",
        "properties": {
          "dhm_u16": {
            "index": 2,
            "texCoord": 0,
            "channels": [0],
            "offset": 0.0,
            "scale": 655.35
          }
        }
      }
    ]
  }
}
```

**Comment by bjornblissing** [comment]
>Multiple channels of a property texture can be used to represent individual bytes of larger data types. The values from the selected channels represent the bytes of the actual property value, in little-endian order.

On its own the above statement is a bit ambiguous IMHO. For example, there is nothing that prevents that from being interpreted as having the individual bytes of two UINT16 values to begin combined into a larger 32 bit value.

However, when we combine that with the requirement that the number of channels should match the number of bytes of the property type it becomes clear that 1 channel equal 1 byte, 2 channels should equal 2 bytes, etc..

It would be better if the requirement for the number of bytes was written more clearly and if the text was kept together so that you don't need to combine several different sentences to derive the final requirement.

My suggestion would have been something like:
> **Implementation note**: Specialized texture formats may allow additional channels, or **channels using a more than 8 bits per pixel**. The usage of such texture formats for property textures has to be defined by additional extensions.

**Comment by javagl** [comment]
I agree that the wording should be clarified. This could/should happen as _part_ of the general clarification about the 16-bit-channels, but maybe a hotfix of changing "higher number" to "more than 8" could be warranted.

A bit more generally: I know that the data with 16bit channels is out there. And I think that the validator _has_ to issue a "WARNING", _unless_ the specification is updated. 

However, I'll leave this open, at least as a reminder to see whether/how the validator could even read/detect 16-bit-channel images with that `sharp` library.
